augly/image/utils/bboxes.py

```diff
@@ -2,7 +2,8 @@
 # Copyright (c) Facebook, Inc. and its affiliates.
 
 import math
-from typing import List, Optional, Tuple
+from PIL import Image, ImageDraw
+from typing import Callable, List, Optional, Tuple
 
 import augly.image.utils as imutils
 import numpy as np
```

Comment:
nit on grouping imports -- the first set of imports should only be imports from python (i.e. no external packages etc)
```suggestion
from typing import Callable, List, Optional, Tuple

import augly.image.utils as imutils
import numpy as np
from PIL import Image, ImageDraw
```